docs/ide/work-with-multi-factor-authentication.md

```diff
@@ -37,20 +37,22 @@ Once the system web browser workflow is enabled, you can sign in or add accounts
 </br>
 :::image type="content" source="media/add-personalization-account.png" alt-text="Add a new personalization account to Visual Studio." border="false":::
 
-This action will open your system's default web browser, ask you to sign into your account, and validate any required MFA policy. 
+This action will open your system's default web browser, ask you to sign into your account, and validate any required MFA policy.
+
+Based on your development activities and resource configuration, a new resource might trigger a new CA/MFA challenge later in the session. If that happens, your account might become stale and you'll need to re-enter your credentials to satisfy that new CA/MFA challenge.
```

Comment:
Change to:

"Based on your development activities and resource configuration, you may be prompted to reenter your credentials during your session. This can occur when you add a new resource, or use a resource with CA/MFA policies that you have not yet validated."

Reply:
Slightly modified your suggestion. Hopefully its a bit more clear now.

Reply:
Should be good to go now. :)